webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -5266,6 +5266,36 @@ var userOverride = {
                         $mck_box_form.addClass('mck-text-req');
                         return false;
                     }
+                    if (
+                        $applozic('#mck-text-box').data('validation') &&
+                        $applozic('#mck-text-box').data('validation') != ''
+                    ) {
+                        var regexForm = $applozic('#mck-text-box').data(
+                            'validation'
+                        );
+                        var fieldValue = message;
```

Comment:
fieldValue is used only once, which means we can avoid this vairable and directly use message inside regexForm.test